lib/timers.js

```diff
@@ -151,11 +151,15 @@ exports._unrefActive = function(item) {
 // Appends a timer onto the end of an existing timers list, or creates a new
 // TimerWrap backed list if one does not already exist for the specified timeout
 // duration.
-function insert(item, unrefed) {
+function insert(item, unrefed, start) {
   const msecs = item._idleTimeout;
   if (msecs < 0 || msecs === undefined) return;
 
-  item._idleStart = TimerWrap.now();
+  if (start !== 0) {
```

Comment:
Can be `typeof start === 'number'` - and just pass in two arguments from `active()` and `unrefActive()`.

Reply:
:+1: